site/en/docs/privacy-sandbox/topics/demo/index.md

```diff
@@ -0,0 +1,120 @@
+
+If no topics have been observed for this hostname and this user, the header is included but the value is empty. In other words, the `Sec-Browsing-Topics` header on a `fetch()` request only includes topics that have been observed for the current user's browser by a caller whose origin matches the hostname of the request URL. This is the same as if you were calling `document.browsingTopics()` from an iframe to view observed topics for the current user.
+
+The request header is sent on a request as long as it has the appropriate permission policy is in play, the context is secure, and user settings permit it. Topics are not provided in headers for navigation requests.
+
+The Topics request header looks like this:
+
+```text
+Sec-Browsing-Topics: 186;version="chrome.1:1:2206021246";config_version="chrome.1";model_version="2206021246";taxonomy_version="1", 265;version="chrome.1:1:2206021246";config_version="chrome.1";model_version="2206021246";taxonomy_version="1"
+```
+
+This example includes two topics from the [Topics taxonomy](https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md), 186 and 265, along with each topic's version information.
+
+{% Aside 'note' %}
+The [fetch()](https://chromium-review.googlesource.com/c/chromium/src/+/4044267) and [XHR](https://chromium-review.googlesource.com/c/chromium/src/+/4103742) implementations were first made available in Chrome 111. (Refer to these builds for more information.)
+{% endAside %}
+
+Inclusion of the topics header in XHR requests is only available temporarily, and support will be removed in the future.
+
+### Mark topics as observed with `Observe-Browsing-Topics`
+
+If a request includes a `Sec-Browsing-Topics` header and the response to that request includes an `Observe-Browsing-Topics: ?1` header, then topics from the request header will be marked by the browser as observed. Observed topics are eligible for calculation by the Topics API. This mechanism is designed to match the functionality provided by using the JavaScript API from an iframe.
+
+The screenshot below shows the topics recorded from visiting the sites on the API demo page.
+
+<figure>
+  {% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/7GjvLNY86mBzeXPERRam.png", alt="Topics API demo page on glitch.me", width="656", height="566" %}
+  <figcaption>The glitch.me demo for trying the API. <a href="https://wd.imgix.net/image/RtQlPaM9wdhEJGVKR8boMPkWf443/7GjvLNY86mBzeXPERRam.png?auto=format&w=1600">View a larger version</a></figcaption>
+</figure>
+
+This list shows the sites you can visit from the demo to record topics of interest. As you can see, the Arts &amp; Entertainment/Humor category in the screenshot is not the topic of one of these websites, so this recorded topic is one that was added as the possible 5 percent random topics.
```

Comment:
@heyawhite Since you're planning for a future change in CMS do you want me to change HTML entities like & in text to the proper html entity &amp; or even the number? Tried it here with no issues.

Reply:
@nancymic2 Yeah, either would work. I appreciate it!

Reply:
wil do later as discussed.